File: POTD/potd(2-10-2023).java
class Solution {
    public boolean winnerOfGame(String colors) {
        int a = 0;
        int b = 0;
        for(int i=1; i<colors.length() -1 ; i++){
            char currentChar = colors.charAt(i);
            
            if (currentChar == 'A' && colors.charAt(i - 1) == 'A' && colors.charAt(i + 1) == 'A') {
                a++;
            } else if (currentChar == 'B' && colors.charAt(i - 1) == 'B' && colors.charAt(i + 1) == 'B') {
                b++;
            }
        }
        return a > b;
    }
}

File: POTD/potd(2-10-2023)/potd(2-10-2023).ts
// POD- Leet Code (2-10-2023)
function winnerOfGame(colors: string): boolean {
    let a:number = 0;
    let b:number = 0;

    for(let i=1; i<colors.length - 1; i++){
        let currChar:string = colors[i];
        if(currChar == 'A' && currChar == colors[i-1] && currChar == colors[i+1]){
            a += 1;
        }else if(currChar == 'B' && currChar == colors[i-1] && currChar == colors[i+1]){
            b += 1;
        }
    }
    console.log(a, b);
    return a > b;
};